(CVE-2012-3401) CVE-2012-3401 libtiff (tiff2pdf):
Heap-based buffer overflow due to improper initialization of
T2P context struct pointer

https://bugzilla.redhat.com/show_bug.cgi?id=837577
---
diff -Naur tiff-4.0.2.orig/tools/tiff2pdf.c tiff-4.0.2/tools/tiff2pdf.c
--- tiff-4.0.2.orig/tools/tiff2pdf.c	2012-06-15 17:51:54.000000000 -0400
+++ tiff-4.0.2/tools/tiff2pdf.c	2012-07-05 13:34:36.569691068 -0400
@@ -1066,6 +1066,7 @@
 				"Can't set directory %u of input file %s", 
 				i,
 				TIFFFileName(input));
+			t2p->t2p_error = T2P_ERR_ERROR;
 			return;
 		}
 		if(TIFFGetField(input, TIFFTAG_PAGENUMBER, &pagen, &paged)){
